src/test/java/uk/gov/hmcts/reform/jobscheduler/jobs/HttpCallJobTest.java

```diff
@@ -41,15 +50,20 @@
     private static final String NEW_S2S_TOKEN = "newly-generated-token";
     private static final String OLD_S2S_TOKEN = "some-token";
     private static final String CUSTOM_VALUE = "anything";
+    private static final String JOB_ID = "jobId123";
 
     @Rule
     public WireMockRule wireMockRule = new WireMockRule();
 
-    @Autowired RestTemplate restTemplate;
+    @Autowired
+    RestTemplate restTemplate;
 
-    @Mock private ActionExtractor actionExtractor;
-    @Mock private JobExecutionContext context;
-    @Mock private AuthTokenGenerator authTokenGenerator;
+    @Mock
+    private ActionExtractor actionExtractor;
+    @Mock
+    private JobExecutionContext context;
+    @Mock
+    private AuthTokenGenerator authTokenGenerator;
```

Comment:
can we keep the previous formatting? :) (unless checkstyle complains)

Reply:
IntelliJ did it. I also think it's better to have field annotations in separate lines, but if it bothers you, I can bring it back to its previous state.